aws/resource_aws_subnet_test.go

```diff
@@ -512,10 +544,8 @@ resource "aws_subnet" "test" {
 func testAccSubnetTagsConfig2(rName, tagKey1, tagValue1, tagKey2, tagValue2 string) string {
 	return fmt.Sprintf(`
 resource "aws_vpc" "test" {
-  cidr_block = "10.1.0.0/16"
-
-  tags = {
-    Name = %[1]q
+  cidr_block = "10.1.0.0/16"tags = {
```

Comment:
Looks like there's a little configuration typo here 😅 

```
=== CONT  TestAccAWSSubnet_tags
TestAccAWSSubnet_tags: resource_aws_subnet_test.go:177: Step 3/4 error: terraform failed: exit status 1
stderr:
Error: Missing newline after argument
on config633927574/terraform_plugin_test.tf line 3, in resource "aws_vpc" "test":
3:   cidr_block = "10.1.0.0/16"tags = {
An argument definition must end with a newline.
```

```suggestion
  cidr_block = "10.1.0.0/16"

  tags = {
```

Reply:
thanks, missed this in the rebase